browserstack.browsers.js

```diff
@@ -1,4 +1,15 @@
 const browserstackBrowsers = {
+  // iOS
+  iphone: {
```

Comment:
I've got a bit of a note in the same file (see https://github.com/skatejs/build/pull/270/files#diff-8033cb64123583879ab30b51a9373f3bR9) but will explain a bit further here:

When we run Karma with heaps of desktop browsers before the mobile browsers, each browser takes up some vertical space inside the viewport (see image below). Because mobile browsers have less vertical space, if too many browsers are listed then the actual live test content/divs are off the bottom of the screen. This seems to prevent some tests from proceeding on iOS/Android (possibly click events).

![image](https://cloud.githubusercontent.com/assets/1089897/17284221/3169c464-57fc-11e6-85e7-33b825af055a.png)